src/components/ViewLink/ViewLink.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import ArrowIcon from '../../assets/ArrowIcon'
+
+const ViewLink = ({ style, link}) => {
+  return (
+    <p
+    style={{
```

Comment:
similar comment as in few other places. better to give it a default style and accept className as props instead. the caller can always add styles the className being passed in props.